### ISSUE TITLE
6.3.1 breaks autosave example

### ISSUE THREAD

**Issue by llamallamagirl** [final-form/react-final-form#677]
### Are you submitting a **bug report** or a **feature request**?
Bug report

### What is the current behavior?

Upon updating to 6.3.1, the AutoSave example no longer works. "values" is always undefined and thus AutoSave never submits form changes.

### What is the expected behavior?

That the AutoSave example would work, or be updated to work.

### Sandbox Link

https://codesandbox.io/s/react-final-form-auto-save-with-debounce-flqx3

### What's your environment?

React-Final-Form: 6.3.1
Final-Form: 4.18.6
React: 16.12.0

**Comment by apete343** [comment]
It looks like a number of the `FormState` values are absent in `6.3.1` vs `6.3.0`.

`6.3.0`
https://codesandbox.io/s/eloquent-feistel-tghxx?fontsize=14

`6.3.1`
https://codesandbox.io/s/affectionate-bird-hp4km?fontsize=14

**Comment by onlyann** [comment]
I think this is to do with the logic in `renderComponent`

@erikras  shouldn't  https://github.com/final-form/react-final-form/blob/32d02b1f37161b939825cf539e873272c0dd4f9c/src/renderComponent.js#L13-L18
be
```
if (component) {
    return React.createElement(
      component,
      Object.assign(lazyProps, rest, {
        children,
        render
      })
    )
  }
``` ?

**Comment by erikras** [comment]
Great reporting, guys! 👍

Fix published in [`v6.3.2`](https://github.com/final-form/react-final-form/releases/tag/v6.3.2).
